core/src/main/java/com/yahoo/imapnio/async/data/QResyncParameter.java

```diff
@@ -0,0 +1,88 @@
+package com.yahoo.imapnio.async.data;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/**
+ * This class models the QRESYNC parameter defined in https://tools.ietf.org/html/rfc7162#page-26.
+ */
+public class QResyncParameter {
+    /** Last known UIDVALIDITY. */
+    private long knownUidValidity;
```

Comment:
Name the variables to match RFC: https://tools.ietf.org/html/rfc7162 page 45
uidValidity
modSeq etc

Reply:
RFC says it is last known uidValidity.
https://tools.ietf.org/html/rfc7162#section-3.2.5

Reply:
Formal Syntax section:
https://tools.ietf.org/html/rfc7162#section-7

select-param        =/  "QRESYNC" SP "(" uidvalidity SP
                       mod-sequence-value [SP known-uids]
                       [SP seq-match-data] ")"
                       ;; Conforms to the generic select-param
                       ;; syntax defined in [RFC4466].